src/plugins/intel_cpu/src/emitters/x64/jit_snippets_emitters.cpp

```diff
@@ -1489,6 +1577,15 @@ void BrgemmCopyBEmitter::execute(matmul::jit_brgemm_matmul_copy_b_t *kernel, con
     (*kernel)(&ctx);
 }
 
+void BrgemmCopyBEmitter::print_debug_info() const {
+    std::cerr << "Emitter type name:" << get_type_name(this) << "\n";
```

Comment:
Can we also store brgemm node (or its name) in the BrgemmCopyBEmitter and print it here?